downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -461,6 +466,123 @@ public function main() returns error? {
 
 To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.9.0%22+label%3AType%2FBug).
 
+### Revamped connector updates
+
+The following new connectors along with major updates have been added to the Ballerina library, as part of the Ballerina connector revamp initiative.
```

Comment:
Should we split these further to new connectors and revamps? May help the descriptions.

```suggestion
New versions of the following connectors have been released with major updates, as part of the Ballerina connector revamp initiative.
```